From f7b2443073519c78492deb425da1f55408900f17 Mon Sep 17 00:00:00 2001
From: StanleyYP Wang <StanleyYP.Wang@mediatek.com>
Date: Thu, 20 Mar 2025 16:41:27 +0800
Subject: [PATCH 2/4] mtk: mt76: mt7996: fix precal size for mt7996 be14000

Fix 6G group precal size for mt7996 BE14000 233
This has been changed in the fw since 2025/3/12

Signed-off-by: StanleyYP Wang <StanleyYP.Wang@mediatek.com>
---
 mt7996/eeprom.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mt7996/eeprom.h b/mt7996/eeprom.h
index 63be9ae4..2370eb0b 100644
--- a/mt7996/eeprom.h
+++ b/mt7996/eeprom.h
@@ -77,7 +77,7 @@ static const u32 mt7996_prek_rev[] = {
 static const u32 mt7996_prek_rev_233[] = {
 	[GROUP_SIZE_2G] =			4 * MT_EE_CAL_UNIT,
 	[GROUP_SIZE_5G] =			44 * MT_EE_CAL_UNIT,
-	[GROUP_SIZE_6G] =			100 * MT_EE_CAL_UNIT,
+	[GROUP_SIZE_6G] =			125 * MT_EE_CAL_UNIT,
 	[ADCDCOC_SIZE_2G] =			4 * 4,
 	[ADCDCOC_SIZE_5G] =			4 * 4,
 	[ADCDCOC_SIZE_6G] =			4 * 5,
-- 
2.45.2

